### ISSUE TITLE
attributes.npy

### ISSUE THREAD

**Issue by DL-Alva** [ylshaooo/keras-zero-shot-detection#4]
Sorry for your time. 

When I try to train this model using my dataset, I change the data classes in file "data/voc_classes.txt". Wether should I change the content of "data/attributes.npy" Correspondingly? However, it seems that I can't find the code which generates the file  "data/attributes.npy".

**Comment by SnowPi** [comment]
I meet the same problem. I want to train this model using my own class, whether should I change the content of "data/attributes.npy"?

**Comment by lil-wayne-0319** [comment]
using some NLP model(https://github.com/stanfordnlp/GloVe) to generate word_vector.txt, and then transfer the .txt to .npy.
but why the loss is keeping Nan. (T _ T)
